dd-sdk-android/src/main/kotlin/com/datadog/android/log/internal/logger/CombinedLogHandler.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-2019 Datadog, Inc.
+ */
+
+package com.datadog.android.log.internal.logger
+
+internal class CombinedLogHandler(
```

Comment:
Maybe WrappedLogHandler ?

Reply:
Wrapped doesn't give the idea that there are multiple handlers wrapped.

Reply:
Yup that's also true, nevermind, to be honest I cannot think to a better name so we could leave it like this.